src/tuntap/windows/c_tuntap_windows.cpp

```diff
@@ -155,6 +195,7 @@ std::wstring c_tuntap_windows_obj::get_device_guid() {
 		try {
```

Comment:
before status = .... reinterpret_cast,  do static_assert tjat LPBYTE as written in comment